src/components/tracked-controls.js

```diff
@@ -18,7 +17,13 @@ module.exports.Component = registerComponent('tracked-controls', {
     controller: {default: 0},
     id: {type: 'string', default: ''},
     idPrefix: {type: 'string', default: ''},
-    rotationOffset: {default: 0}
+    rotationOffset: {default: 0},
+    // Arm model parameters, to use when not 6DOF. (pose hasPosition false, no position)
+    headElement: {type: 'selector'},
+    hand: {type: 'string', default: 'right'},
```

Comment:
it is really only the arm-model that needs to know which hand to use (or none) - it is intended to be passthrough like rotationOffset for hand-controls.  other ways to specify arm-model?  not sure 3dof-controls is a good idea since you have to modify recorders, teleporters, etc. ... but maybe if the arm model can be a separate component (order of operations problem when updating pose though) and tracked-controls still holds the resultant position and rotation.